src/screens/internal/ShareImage/ChartShareImage.tsx

```diff
@@ -12,29 +12,24 @@ import {
 import { DarkScreenshotWrapper } from './ShareImage.style';
 import LogoDark from 'assets/images/logoDark';
 import { chartDarkMode } from 'assets/theme/palette';
-import { Projections } from 'common/models/Projections';
 import { MetricChart } from '../../../components/Charts';
 import { ALL_METRICS, getMetricNameExtended } from 'common/metric';
 import { Metric } from 'common/metric';
-import { findCountyByFips } from 'common/locations';
-import { useProjections } from 'common/utils/model';
-import { Projection } from 'common/models/Projection';
+import { useProjectionsFromRegion } from 'common/utils/model';
 import { SCREENSHOT_CLASS } from 'components/Screenshot';
+import { useRegionFromParams } from 'common/regions/region_hooks';
```

Comment:
It's better to import from `common/regions`, that way we don't leak implementation details of the module (if we decide to move the function, for example)